contracts/Airdrop.sol

```diff
@@ -0,0 +1,177 @@
+/* solhint-disable-next-line compiler-fixed */
+pragma solidity ^0.4.17;
+
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+// Utility chain: Airdrop
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+import "./Workers.sol";
+import "./Pricer.sol";
+
+
+contract Airdrop is Pricer, Workers {
+
+    /*
+     *  Events
+    */
+    /// Event for AirdropPayment complete
+    event AirdropPayment(
```

Comment:
Indexing is cool. Does indexing cost extra gas or it's neglegible?
Can we do indexing on uint256 also?

Reply:
also for formatting on events we've previously filled the line (different from function signatures)
Didn't understand this point. Please explain with an example?

Reply:
@benjaminbollen: for consistency/ease, we started declaring events similar to functions,
```
event EventName(
arg,
arg,
arg)
```
This way it not subject to personal preference in terms of how many is too many arguments to list on the line. We can go back to the other way, if preferred, but that will mean changes in other places (to be consistent); if we do go back to the previous style, we should agree a consistent max number of arguments per line (so that how it looks in an individual dev's editor is not driving the decision).